nengo_loihi/builder/connection.py

```diff
@@ -282,6 +282,93 @@ def build_host_to_learning_rule(model, conn):
     nengo_model.build(pre2send)
 
 
+def build_conv2d_connection(model, conn):
```

Comment:
I would probably put it after `build_chip_connection`, since it's a subset of that, but not a big deal either way.